### ISSUE TITLE
3.2.5: Change on Request Understanding update

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#738]
https://www.w3.org/WAI/WCAG21/Understanding/change-on-request.html#benefits

The first benefit for 3.2.5 is:

Benefits

> - Individuals who are unable to detect changes of context or may not realize that the context has changed are less likely to become disoriented while navigating a site. For example:
>            - individuals who are blind or have low vision may have difficulty knowing when a visual context change has occurred, such as a new window popping up. In this case, warning users of context changes in advance minimizes confusion when the user discovers that the back button no longer behaves as expected.

The text "are less likely to become disoriented" must be "are more likely to become disoriented"

**Comment by patrickhlauke** [comment]
I think it's correct as it currently is. This explains the benefit of following 3.2.5 for users. However, I can see how this could be slightly confusing, as you could also read it as "in general, these users are more confused. so, doing this will make them less confused"

**Comment by jake-abma** [comment]
I see, you can read it both ways. Relating to the title (way above) it makes sense, not sure if this relationship will always be made (not as stand alone / by me apparently :-) )

Making the sentence independent clear we may add: (**if the change is initiated by the user**)

So, instead of:

> Individuals who are unable to detect changes of context or may not realize that the context has changed are less likely to become disoriented while navigating a site. For example:

Make it:

> Individuals who are unable to detect changes of context or may not realize that the context has changed are less likely to become disoriented if the change is initiated by the user while navigating a site. For example:

**Comment by alastc** [comment]
How about:
> Individuals who are unable to detect changes of context, or may not realize that the context has changed if they did not initiate the change are less likely to become disoriented while navigating a site.

In PR #754

**Comment by jake-abma** [comment]
Hi @alastc 

> Individuals who are unable to detect changes of context, or may not realize that the context has changed if they did not initiate the change are less likely to become disoriented while navigating a site.

This one has exactly the same problem, you tell here "people unable to detect the change are less likely to become disoriented" and "people not realizing that the context has changed  are less likely to become disoriented" While it is more likely (if yout not checking the SC normative text at the top.

The part "less likely disoriented if they make the change themselves" is the key here

**Comment by jake-abma** [comment]
@alastc any feedback on my last comment? Any progress for this one in near future? Thx!

**Comment by bruce-usab** [comment]
Discussed on backlog call today.  See PR, but TF settled on:

> Individuals who may not realize that the context has changed if they did not initiate the change themselves are less likely to become disoriented while navigating a site.
